server/index.js

```diff
@@ -1,9 +1,32 @@
 const express = require('express');
 const path = require('path');
+const bodyparser = require('body-parser');
 const db = require('../data/index');
 
+
 const app = express();
 
 const port = process.env.PORT || 3000;
 
+app.use('/rooms/:room_id', express.static(path.join(__dirname, '..', 'client', 'dist')));
+
+app.use(bodyparser.json());
+
+app.get('/rooms/:room_id/reservations', (req, res) => {
+  // db.Calendar.hasMany(db.Room, { foreignKey: 'id' });
+  // db.Room.belongsTo(db.Calendar, { foreignKey: 'id' });
+  // db.Calendar.findAll({
+  //   where: { room_id: req.params.room_id },
+  //   include: [{
+  //     model: db.Room,
+  //     required: false,
+  //     where: { id: req.params.room_id },
+  //   }],
+  // });
```

Comment:
@Pfdiamond13 @fradleybrancis Remove "cruft" before merging into master. If the feature is incomplete, it should live in a feature branch until it's ready.